AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.swift

```diff
@@ -83,56 +83,56 @@ let package = Package(
                 .awsSDKHTTPAuth,
                 .awsSDKIdentity
             ],
-            path: "Sources/Core/AWSClientRuntime/Sources",
+            path: "Sources/Core/aws-sdk-swift.AWSClientRuntime/Sources",
```

Comment:
Client directories have changed (yes, again, sorry.)

Changes in this file are to get pointed to the new directories.